sql/table.go

```diff
@@ -87,13 +91,21 @@ func makeTableDesc(p *parser.CreateTable) (structured.TableDescriptor, error) {
 				ColumnNames: d.Columns,
 			}
 			if d.PrimaryKey {
-				index.Name = structured.PrimaryKeyIndexName
+				if index.Name == "" {
```

Comment:
can you add a comment here to explain why this is not symmetrical with the above case?